browser/mojSortableTable.ts

```diff
@@ -252,8 +299,22 @@ $(() => {
 
   const table = tables[0]
 
+  class SecondOrderSortableTable extends MOJFrontend.SortableTable {
```

Comment:
Is it possible to move this out of the jquery function to stick it with the other class you've added, or is there a reason for it being here?